### PR TITLE
doing homework3

### PR DESCRIPTION
**Pull request by Hromy-Ann**
task2 is given by two variants

### CODE REVIEW

File: src/main/java/org/example/homework3/Task2.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package org.example.homework3;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int a = 210;
+        int b = 15;
+        int min;
+        int max;
+        if (a > b) {
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Task2.java]: Try to use the ternary operator here for defining min and max.
Solution is absolutely correct, but we didn't learn if-else construction, yet

File: src/main/java/org/example/homework3/Task3.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.example.homework3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        long a = 12345678912345L;
+        int b = ((int) a);
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Task3.java]: unnecessary parentheses
```suggestion
        int b = (int) a;

```

File: src/main/java/org/example/homework3/Task5.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package org.example.homework3;
+
+public class Task5 {
+    public static void main(String[] args) {
+        double celsius = 36.6;
+        double fahrenheit = celsius * 9 / 5 + 32;
+        double kelvin = celsius + 273;
+        System.out.printf("%.1f °C = %.2f °F\n", celsius, fahrenheit);
+        System.out.println(celsius + " °C = " + kelvin + "K");
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Task5.java:9]: Nice output :)

**Review by alexshamrai — APPROVED** [review]

File: src/main/java/org/example/homework3/Task2.java (hunk 1 above)
Comment (Hromy-Ann) [reply src/main/java/org/example/homework3/Task2.java]: I have refactored task2 using the ternary operator. I have squashed upates into one commit.

Comment (alexshamrai) [reply src/main/java/org/example/homework3/Task2.java]: Feel free to merge, PR is already approved

**Pull request merged**